variables.tf

```diff
@@ -0,0 +1,48 @@
+variable "app_name" {
```

Comment:
use standard `name` variable for interoperability with our other modules. Also add `namespace`